### DISCUSSION TITLE
fatal error: 'dispatch/dispatch.h' file not found

### DISCUSSION THREAD

**Discussion by jarvys** [zed-industries/zed#7561] [Feature Requests]
`generate_dispatch_bindings` panic when I build zed. How can I fix this error? 

Sonoma 14.2
Xcode 15.2

**Comment by cyypherus** [comment]
Try running this before `cargo run`
`export CPATH=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include`
[From this stackoverflow answer](https://stackoverflow.com/questions/58278260/cant-compile-a-c-program-on-a-mac-after-upgrading-to-catalina-10-15)

**Reply by waterblower** [reply]
it works

**Comment by mocenigo** [comment]
It worked today for me because I was getting this very error. I have added the line to the bundle-mac script.

**Comment by warp-records** [comment]
worked for me too

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
